src/features/commentsSlice.ts

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import * as commentsApi from '../api/comments';
+import { Comment } from '../types/Comment';
+
+export interface CommentsState {
+  comments: Comment[],
+  loaded: boolean,
+  hasError: boolean,
+}
+
+const initialState: CommentsState = {
+  comments: [],
+  loaded: false,
+  hasError: false,
+};
+
+export const getCommentsAsync = createAsyncThunk(
+  'comments/getComments',
+  async (postId: number) => {
+    const value = await commentsApi.getPostComments(postId);
+
+    return value;
+  },
```

Comment:
```suggestion
    return commentsApi.getPostComments(postId);
  },
```
just FYI - it will work in the same way